starter-code/server.js

```diff
@@ -67,20 +82,30 @@ app.post('/articles', function(request, response) {
 });
 
 app.put('/articles/:id', function(request, response) {
-  // TODO: Write a SQL query to update an author record. Remember that our articles now have
+  // TO/DO: Write a SQL query to update an author record. Remember that our articles now have
   // an author_id property, so we can reference it from the request.body.
-  // TODO: Add the required values from the request as data for the SQL query to interpolate
+  // TO/DO: Add the required values from the request as data for the SQL query to interpolate
   client.query(
-    ``,
-    []
+    `UPDATE authors SET author_id = $1, author = $2, "authorUrl" = $3 WHERE author_id = $1`,
```

Comment:
Not a big deal, but you don't need to set `author_id = $1` when the `WHERE` clause already asserts that they match.